ext/SConscript

```diff
@@ -37,16 +37,24 @@ def prep_gmock(env):
 def prep_fmt(env):
     localenv = prep_default(env)
     if not env['system_fmt']:
+        if LooseVersion(env['FMT_VERSION']) >= LooseVersion('5.0.0'):
+            localenv.Prepend(CPPPATH=Dir('#include/cantera/ext'))
         license_files.append(('fmtlib', 'fmt/LICENSE.rst'))
-        for name in ('format.h', 'ostream.h'):
+        for name in ('format.h', 'ostream.h', 'printf.h', 'core.h', 'format-inl.h', 'posix.h'):
             build(copyenv.Command("#include/cantera/ext/fmt/" + name,
-                                  "#ext/fmt/fmt/" + name,
+                                  "#ext/fmt/include/fmt/" + name,
                                   Copy('$TARGET', '$SOURCE')))
     return localenv
 
 # each element of libs is: (subdir, (file extensions), prepfunction)
-libs = [('libexecstream', ['cpp'], prep_default),
-        ('fmt/fmt', ['cc'], prep_fmt)]
+libs = [('libexecstream', ['cpp'], prep_default)]
+
+# fmtlib versions less than 5.0.0 were not distributed with a compiled library,
+# so we should build it ourselves. Otherwise, only need to build if not using the system fmt
+if LooseVersion(env['FMT_VERSION']) < LooseVersion('5.0.0'):
```

Comment:
Distributed by whom? I think the point is, if we are using a non-system `fmt`, we want to compile the source files (which are, of course, in a different directory in 5.0.0). And I guess, in the case of the system `fmt` library, we are expecting to find a compiled library to link to.